clarify steps for MySQL config. (#12759)
DataDog/integrations-core: mysql/README.md

The following instructions grant the Agent permission to login from any host using `datadog@'%'`. You can restrict the `datadog` user to be allowed to login only from localhost by using `datadog@'localhost'`. See [MySQL Adding Accounts, Assigning Privileges, and Dropping Accounts][5] for more info.

> For MySQL up to version 7.x create the `datadog` user this way:

Reviewer: ```suggestion
For MySQL up to version 7.x, create the `datadog` user with the following command:
```

> For MySQL up to version 7.x, create the `datadog` user with the following command:

Reviewer 2: ```suggestion
For MySQL versions 5.6 and 5.7, create the `datadog` user with the following command:
```

```shell
mysql> CREATE USER 'datadog'@'%' IDENTIFIED BY '<UNIQUEPASSWORD>';
Query OK, 0 rows affected (0.00 sec)
```

> For MySQL 8.0+ create the `datadog` user with the native password hashing method:

Reviewer: ```suggestion
For MySQL 8.0 or greater, create the `datadog` user with the native password hashing method:
```

echo -e "\033[0;31mCannot connect to MySQL\033[0m"
```

> The Agent needs a few privileges to collect metrics. Grant the `datadog` user the following limited privileges ONLY.

Reviewer: ```suggestion
The Agent needs a few privileges to collect metrics. Grant the `datadog` user only the following limited privileges.
```

> For MySQL up to version 7.x set `max_user_connections` this way:

Reviewer: ```suggestion
For MySQL up to version 7.x, set `max_user_connections` with the following command:
```

> For MySQL up to version 7.x, set `max_user_connections` with the following command:

Reviewer 2: ```suggestion
For MySQL versions 5.6 and 5.7, set `max_user_connections` with the following command:
```

```shell
mysql> GRANT REPLICATION CLIENT ON *.* TO 'datadog'@'%' WITH MAX_USER_CONNECTIONS 5;
Query OK, 0 rows affected, 1 warning (0.00 sec)
```

> For MySQL 8.0+ set `max_user_connections` this way:

Reviewer: ```suggestion
For MySQL 8.0 or greater, set `max_user_connections` with the following command:
```
